drivers/sensor/sgp40/sgp40.c

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (c) 2021 Leonard Pollak
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sensirion_sgp40
```

Comment:
How was this tested? I don't see a devicetree binding here.  There should be one.

Reply:
I have added a devicetree binding since I had one setting as Kconfig option.